operations/serviceoverbookinglimits.md

```diff
@@ -0,0 +1,208 @@
+
+It's not allowed to update past service overbooking limits outside of `EditableHistoryInterval`.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/serviceOverbookingLimits/set`
+
+```javascript
+{
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0",
+  "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+  "SetServiceOverbookingLimits": [
+    {
+      "FirstTimeUnitStartUtc": "2024-11-01T00:00:00Z",
+      "LastTimeUnitStartUtc": "2024-11-30T00:00:00Z",
+      "Limit": 10
+    }
+  ],
+  "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `EnterpriseId` | string | optional | Unique identifier of the enterprise. Required when using [Portfolio Access Tokens](../guidelines/multi-property.md), ignored otherwise. |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) overbooking limits will be set in. |
+| `SetServiceOverbookingLimits` | array of [Service overbooking limits set parameters](serviceoverbookinglimits.md#service-overbooking-limits-set-parameters) | required, max 1000 items | Collection of service overbooking limits to be set. |
+
+#### Service overbooking limits set parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's [time unit](../concepts/time-units.md): 367 days. |
+| `Limit` | integer | required | Number of overbookings allowed for the `Service`. Must be non-negative. |
+
+### Response
+
+```javascript
+{}
+```
+
+## Clear service overbooking limits
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change.
+
+Deletes service overbooking limits that exactly match the specified conditions. This operation is intended to be used alongside `serviceOverbookingLimits/set`. The specified conditions must be met exactly. The time interval, however, does not need to correspond to an existing service overbooking limit in the system, instead the API uses a splicing algorithm to work out how to divide up any existing service overbooking limit to meet the specified time interval.
```

Comment:
Same suggestion as above, I prefer a single paragraph for the description. And same comment about `EditableHistoryWindow`. Otherwise looks good.